pkg/controller.v1/common/pod.go

```diff
@@ -484,6 +490,11 @@ func (jc *JobController) createNewPod(job interface{}, rt apiv1.ReplicaType, ind
 		}
 	}
 
+	// Creation is expected when there is no error returned
+	// We use `RaiseExpectations` here to accumulate expectations since `SetExpectations` has no such kind of ability
+	expectationPodsKey := expectation.GenExpectationPodsKey(jobKey, rt)
+	jc.Expectations.RaiseExpectations(expectationPodsKey, 1, 0)
```

Comment:
Seems arguments to `RaiseExpectation` are all 1, 0 or 0, 1. Why not to use `ExpectCreations` directly?

Reply:
Same question here.

Reply:
`ExpectCreations(key, 1)` calls `SetExpectations` underneath, it will set "expectedCreation" to `1` instead of adding(`expectedCreation += 1`). We are here to expect multiple creations, am i right?

Reply:
Gotcha. SGTM.

Reply:
`expectationPodsKey` represents a group of pods. It does expect multiple creations.